seatunnel-shade/seatunnel-hadoop3-3.1.4-uber/pom.xml

```diff
@@ -52,6 +52,11 @@
             <artifactId>snappy-java</artifactId>
             <version>1.1.10.4</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-aws</artifactId>
+            <version>${hadoop3.version}</version>
+        </dependency>
```

Comment:
cc @EricJoy2048

Reply:
Why add `hadoop-aws` to hadoop shade jar?

Reply:
Changed to not add `hadoop-aws`.